src/System.Management.Automation/resources/ParameterBinderStrings.resx

```diff
@@ -142,7 +142,7 @@
     <value>Cannot bind positional parameters because no names were given.</value>
   </data>
   <data name="AmbiguousParameterSet" xml:space="preserve">
-    <value>Parameter set cannot be resolved using the specified named parameters.</value>
+    <value>Parameter set cannot be resolved using the specified named parameters. One or more of the parameters issued cannot be used together. Do a Get-Help on your script/function/cmdlet for the syntax help.</value>
```

Comment:
* `One or more of the parameters issued cannot be used together.` This still doesn't address the case where too few parameters are supplied. For example: `foo` has 2 parameter sets `-A -B -C` and `-A -B -D` but the user only supplied `-A -B`. It's not that `-A` and `-B` can't be used together, but that `-C` or `-D` are needed to determine the parameter set.
* `Do a Get-Help on your script/function/cmdlet for the syntax help.`  I think this is would be somewhat confusing. I'm not sure how to fix it. The wording could at least be improved `Use Get-Help with your script, function, or cmdlet for syntax help.`